packages/api/tests/endpoints/test-granules.js

```diff
@@ -628,19 +825,32 @@ test('DELETE deleting an existing unpublished granule', async (t) => {
 });
 
 test('DELETE for a granule with a file not present in S3 succeeds', async (t) => {
```

Comment:
This is the test I was referring to above. In our Granule DELETE flow we actually try to delete S3 files twice. Once in the PG model and once in the Dynamo model. I left it because my understanding is that the Dynamo model will eventually be deprecated and it doesn't appear to hurt to just try to delete already deleted files.

Open to removing it from the Dynamo model if that feels safe. I just got squeamish.

Reply:
Discussed in architecture meeting. We're splitting out S3 deletes now so that they're handled outside of the PG File model. The Dynamo model will still try to delete from S3 again but does not cause problems if there's nothing to delete.